app/scripts/views/mixins/password-mixin.js

```diff
@@ -51,8 +51,8 @@ define([
           this.logScreenEvent('password.hidden');
         }
       } catch(e) {
-        // IE8 blows up when changing the type of the input field. Other
```

Comment:
Can you keep the original comment so we have context if we ever add IE8 support back?